### PR TITLE
60 prettify exdb project

### PR DESCRIPTION
**Pull request by tom08**

### CODE REVIEW

File: exdb/models.py [hunk 1]
```diff
@@ -56,12 +59,12 @@ def __str__(self):
 
 class Experience(models.Model):
     STATUS_TYPES = (
```
Comment (shaneallgeier) [inline exdb/models.py]: We should include a comment about what the third value in the tuple is for.

File: exdb/templates/exdb/base.html [hunk 2]
```diff
@@ -21,40 +21,33 @@
...
+                <div class="main-menu {% block header_color %}menu-color{% endblock %}">
+                    <div class="title-bar" data-responsive-toggle="header-menu" data-hide-for="medium">
+                        <button class="menu-icon" type="button" data-toggle></button>
+                        <div class="title-bar-title">Menu</div>
+                    </div>
+
+                    <div class="top-bar" id="header-menu">
+                        <div class="top-bar-left">
+                            <ul class="vertical medium-horizontal menu" data-dropdown-menu>
+                                <li class="menu-text"><h2>{% trans 'Hello,' %} {{ user }}</h2></li>
```
Comment (enki211) [inline exdb/templates/exdb/base.html]: We should probably use user.first_name to make it more user friendly

File: exdb/views.py [hunk 3]
```diff
@@ -241,6 +242,63 @@ def form_valid(self, form):
...
+        time_ahead += settings.HALLSTAFF_UPCOMING_TIMEDELTA if self.request.user.is_hallstaff() else settings.RA_UPCOMING_TIMEDELTA
+        Qs = Q(status='ad') & Q(start_datetime__gt=timezone.now()) & Q(start_datetime__lt=time_ahead)
+        user_Qs = Q(author=self.request.user) | Q(planners=self.request.user)
+        if self.request.user.is_hallstaff():
+            user_Qs = user_Qs | Q(recognition__affiliation=self.request.user.affiliation)
+        Qs = Qs & user_Qs
+        return Experience.objects.filter(Qs).distinct().order_by('start_datetime')
+
+    def status_queryset(self):
+        Qs = Q(author=self.request.user) | (Q(planners=self.request.user) & ~Q(status='dr'))
```
Comment (enki211) [inline exdb/views.py]: Should probably add Q(next_approver=self.request.user)

**Comment by tom08** [comment]
I approve of the parts I did not write.

**Comment by shaneallgeier** [comment]
We still need to steal the [official-looking login page](https://mis.missouristate.edu/BearPassCard/
) from the MIS department.

**Comment by enki211** [comment]
That is a very strong word. How about borrow without asking?

File: exdb/templates/exdb/experience_view.html [hunk 4]
```diff
@@ -13,8 +13,8 @@
 
 {% block content %}
     <div class="row">
-        <div class="small-3 small-centered columns">
-            <h1>{{ experience.name }}</h1>
+        <div class="small-12 small-centered columns">
+            <h1 class="center">{{ experience.name }}</h1>
```
Comment (tom08) [inline exdb/templates/exdb/experience_view.html]: Foundation has a helper class "text-center" that does the same thing as this class.

File: exdb/templates/exdb/experience_view.html [hunk 5]
```diff
@@ -5,11 +5,18 @@
...
+{% endblock %}
+
 {% block content %}
-    <p><a href="{% url 'home' %}">{% trans "Return Home" %}</a></p>
-    <h1>{{ experience.name }}</h1>
-    <hr/>
-    <div class="experience-display">
+    <div class="row">
+        <div class="small-12 small-centered columns">
+            <h1 class="center">{{ experience.name }}</h1>
```
Comment (tom08) [inline exdb/templates/exdb/experience_view.html]: Foundation has a helper class "text-center" that does the same as this "center" class.

File: exdb/templates/exdb/base.html [hunk 6]
```diff
@@ -1,36 +1,62 @@
...
-                <h2>{% trans 'Hello' %} {{ user }}</h2>
-                <div>
-                    <form method="GET" action="{% url 'search' %}">
-                        <input type="text" name="search" placeholder="{% trans 'Search Experiences' %}" value="{{ search_query }}" />
-                        <input type="submit" value="{% trans 'Search' %}" />
-                    </form>
+                <div class="main-menu {% block header_color %}menu-color{% endblock %}">
+                    <div class="title-bar" data-responsive-toggle="header-menu" data-hide-for="medium">
+                        <button class="menu-icon" type="button" data-toggle></button>
+                        <div class="title-bar-title">Menu</div>
```
Comment (shaneallgeier) [inline exdb/templates/exdb/base.html]: This needs `{% trans %}` tags (and I think it's my fault).

File: exdb/templates/exdb/experience_approval.html [hunk 7]
```diff
@@ -4,44 +4,77 @@
...
+    <div class="row">
+        <div class="medium-7 columns">
+            <form method="POST">
+                {% csrf_token %}
+                <fieldset class="fieldset">
+
+                    <div class="row">
+                        <div class="text-center">
+                            <strong>
+                                Set the supervisor field to the appropriate user if this experience needs LLC approval.
```
Comment (shaneallgeier) [inline exdb/templates/exdb/experience_approval.html]: This needs {% trans %} tags.

File: exdb/templates/exdb/home.html [hunk 8]
```diff
@@ -8,43 +8,57 @@
...
+                                <h3>{{ status }}</h3>
+                                {% for experience in experiences %}
+                                    <a href="{% experience_url experience user %}">
+                                        <div class="{{ status|cut:" " }}-background">
+                                            {{ experience.name }} {{ experience.start_datetime|date:"DATE_FORMAT" }}
+                                        </div>
+                                    </a>
+                                {% endfor %}
+                                <div class="more">
+                                {% if status == 'Upcoming' %}
```
Comment (enki211) [inline exdb/templates/exdb/home.html]: Indentation

**Comment by tom08** [comment]
I approve

**Pull request merged**